app-mock/src/androidTest/java/pl/dybczak/githubusers/robots/SearchPageRobot.kt

```diff
@@ -0,0 +1,52 @@
+package pl.dybczak.githubusers.robots
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.typeText
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.*
+import org.hamcrest.CoreMatchers.allOf
+import pl.dybczak.githubusers.utils.noViewMatcher
+import pl.sulej.users.R
+
+class SearchPageRobot {
```

Comment:
```suggestion
class UsersPageRobot {
```

As suggested in comment in test class some methods could be made public so that test class would operate on them.